ui/plugin-system/src/model/thresholds.ts

```diff
@@ -0,0 +1,12 @@
+export interface StepOptions {
+  value: number;
+  color?: string;
+  name?: string;
+}
+
+export interface ThresholdOptions {
+  mode?: 'percentage' | 'absolute';
```

Comment:
Debating if i should make this a boolean instead. 

For now the default is 'absolute'

Reply:
I think not doing boolean in this case is okay since the strings are more descriptive. Main small thing is we've been using PascalCase for enum values like this and maybe just 'Percent' instead of 'Percentage'?